core/src/main/java/com/klaviyo/coresdk/networking/KlaviyoEvent.kt

```diff
@@ -0,0 +1,7 @@
+package com.klaviyo.coresdk.networking
+
+sealed class KlaviyoEvent(val name: String) {
```

Comment:
Note to self: Fill this out with more Klaviyo events

Reply:
I'm a little curious whether adding non-custom events here could be confusing, for example say the customer had a shopify store integrated which tracks a "Checkout Started" event, and then they start tracking one here (which would show up as a custom event). I just want to make sure that product is on board with potentially confusing scenarios like that.